packages/redux-store/src/auth/TeamState/saga.ts

```diff
@@ -306,7 +308,7 @@ function* addUserMock({ payload }: PayloadAction<WithCallback<IAddUser, ITeamUse
   const teamState: TeamState = yield select();
   yield put(actions.setTeamState({ addUserDialogState: { ...teamState.addUserDialogState, loading: true } }));
   yield delay();
-  const newUser: ITeamUser = { ...body, ...userTeamDemo };
+  const newUser: ITeamUser = { ...body, ...userTeamDemo, id: `${uuid()}` };
```

Comment:
move userTeamDemo to the first entry  to make it default properties { ...userTeamDemo, ...body,  id: `${uuid()}` }